Makefile

```diff
@@ -1,11 +1,23 @@
 SOURCE_FILES := $(shell find . -type f -name '*.go')
+VERSION := $(shell git describe --tags --abbrev=0 | cut -c2-)
 
 policy.wasm: $(SOURCE_FILES) go.mod go.sum
-	docker run --rm -v ${PWD}:/src -w /src tinygo/tinygo:0.18.0 tinygo build \
-		-o policy.wasm -target=wasi -no-debug .
+	docker run \
+		--rm \
+		-e GOFLAGS="-buildvcs=false" \
+		-v ${PWD}:/src \
+		-w /src tinygo/tinygo:0.23.0 \
+		tinygo build -o policy.wasm -target=wasi -no-debug .
 
-annotated-policy.wasm: policy.wasm metadata.yml
-	kwctl annotate -m metadata.yml -o annotated-policy.wasm policy.wasm
+artifacthub-pkg.yml: metadata.yml go.mod
+	kwctl scaffold artifacthub \
+	    --metadata-path metadata.yml --version $(VERSION) \
+		--questions-path questions-ui.yml > artifacthub-pkg.yml.tmp \
+	&& mv artifacthub-pkg.yml.tmp artifacthub-pkg.yml \
+	|| rm -f artifacthub-pkg.yml.tmp
```

Comment:
```suggestion
        $(warning If you are updating the artifacthub-pkg.yml file for a release, \
                remember to set the VERSION variable with the proper value. \
                To use the latest tag, use the following command:  \
                make VERSION=$$(git describe --tags --abbrev=0 | cut -c2-) annotated-policy.wasm)
        kwctl scaffold artifacthub \
            --metadata-path metadata.yml --version $(VERSION) \
                --questions-path questions-ui.yml --output artifacthub-pkg.yml 
```

Reply:
done in https://github.com/kubewarden/hostpaths-psp-policy/pull/14/commits/1fe372d56cf24a2234f517c565170aee49363ae9.